Will Heikoff Documentation

```diff
@@ -1,13 +1,34 @@
-Found zip code file for line 22: ZIPS.shp
+Will Heikoff Documentation
+Goal: To get the python script K-function Local, G:\MAX-Filer\Research\Labs-kbuzard-S18\kfunctions\K-function Local, for California R & D Labs running for 9 iterations
+
+6/12/18
+Searched for file which exhibits the zip code with the associated Total and manufacturing employment for the zip code.
+
+6/15/18
+Found zip code file for line 22: ZIPS.shp, using trial and error methods testing different files with the name ‘zips’ within the California folder, plugging the files into the ArcMap file ‘Base’, located within the same folder, to find a file which exhibits the zip code with the associated Total and manufacturing employment for the zip code.
+	Located at G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data
 Entered ZIPS.shp into line 22
-Found labs data for line 23: Labs.shp
+Found labs data for line 23: Labs.shp, using trial and error methods testing different files with the name ‘zips’ within the California folder, plugging the files into the ArcMap file ‘Base’, located within the same folder, to find a file which exhibits the Labs street address and associated ‘X’ and ‘Y’ cooridinates
```

Comment:
Why "labs" instead of "good labs"? The two shapefiles have different numbers of entries. Good labs has fewer labs. We don't have documentation in this folder to tell us why some were taken out, so we need to figure out what's going on. First task: check the JUE paper to see how many labs it says there are in California and see if it matches up with one of these files. We can talk on Monday afternoon about where to go from there.